### PR TITLE
Add Meta Description for Site Homepage

### LINKED ISSUE #110: Meta Description is Missing
The meta description is broken for the homepage and the state pages, as I can see via Duck Duck Go or Google:

screenshot_20220602-170407

We should make the meta description for the state page unique to the state

### PR DESCRIPTION
**Pull request by herrerarobert**
## Overview
I have updated the Homepage to contain a description using Helmetjs SEO tag. This resolves the issue of search engine result pages not having a "proper" description.

Closes #110

### Demo

image

### Notes

If the current description is not clear enough for the site, let me know and I can change the exact wording.

## Testing Instructions
This was tested using: https://metatags.io/

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -48,8 +48,9 @@ const IndexPage = ({data}) => {
   
   return (
     <Layout>
-      <SEO title="What does it take to decarbonize your state?" />
-
+      <SEO title="What does it take to decarbonize your state?" 
+        description={`To get to zero by 2050, the US must cut climate pollution by ${cutPerYearPrcnt}% a year.`}
```
Comment (vkoves) [inline src/pages/index.js:52]: @derekeder - do you have any suggestions on wording for the site description?

My thought though:

> You can help decarbonize your state, and it's a lot simpler than you think! Learn how with Decarbonize My State.

**Review by vkoves — CHANGES_REQUESTED** [review]
Few minor comments, but looks great! Thanks for checking and confirming the state pages already have a meta description

File: src/components/state-details.js [hunk 2]
```diff
@@ -257,6 +257,7 @@ export default function StateDetailsPage ({ location, data }) {
         title={`What does it take to decarbonize ${placeTitle}?`}
         description={descriptionText}
       />
+      <meta name="description" content="State Meta Tag Test"></meta>
```
[the remark this answers is not in the document]
Comment (herrerarobert) [reply src/components/state-details.js]: I have removed this tag.

File: src/pages/index.js (hunk 1 above)
Comment (herrerarobert) [reply src/pages/index.js]: I have updated the copy. Can change it again if we get more feedback.

**Review by vkoves — APPROVED** [review]
Looks great, awesome work!

**Pull request merged**